contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -113,13 +114,21 @@ contract BiconomyTokenPaymaster is
         address indexed sender,
         address indexed token,
         uint256 indexed totalCharge,
-        address oracleAggregator,
         uint32 priceMarkup,
         bytes32 userOpHash,
-        uint256 exchangeRate,
+        uint128 exchangeRate,
         ExchangeRateSource priceSource
```

Comment:
Can we please move these to the interface?

Reply:
ok